src/js/modules/infragistics.ui.editors.js

```diff
@@ -4808,6 +4840,9 @@
 			}
 			return x;
 		},
+		_roundExponential: function(value, decimals) {
```

Comment:
We discussed this is done due to `toFixed()` rounding, however I found the decimal rounds like [`Math.round10`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/round#Decimal_rounding) in our util file - seems to do exactly what you need, so maybe use that instead? :)
P.S. Can you also copy the description from MDN and add a link where it came from in util?